firefox_ui_tests/remote/security/test_green_larry.py

```diff
@@ -0,0 +1,109 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestGreenLarry(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.url = 'https://ssl-ev.mozqa.com/'
+
+        self.identity_popup = self.browser.navbar.locationbar.identity_popup
+
+    def tearDown(self):
+        try:
+            # TODO: PR 105/115: insure identity_popup closes
```

Comment:
I'll look for that update tomorrow morning. 

I should be able to finish it up if others don't get to it.